src/components/orders/UpdateOrder/UpdateOrder.tsx

```diff
@@ -41,9 +42,9 @@ const validationSchema =  yup.object({
 
 const UpdateOrder: FC<IProps> = ({ open, handleClose, order, setOrder }) => {
 
-  const [error,setError] = useState<string>('');
+  const [error, setError] = useState<string>('');
```

Comment:
I think it's not necessary to use typing, it should set automatically, when you define error in state as an empty string.
```suggestion
  const [error, setError] = useState('');
```